tcms/testruns/admin.py

```diff
@@ -30,9 +31,18 @@ def delete_view(self, request, object_id, extra_context=None):
         return HttpResponseRedirect(reverse('testruns-get', args=[object_id]))
 
 
+class TestExecutionStatusAdminForm(forms.ModelForm):
+    name = forms.CharField(help_text=_("""
+        For more information about customizing TestExecutionStatus see
+        <a href="https://kiwitcms.readthedocs.io/en/latest/admin.html#test-execution-statuses">
+        the documentation</a>
+        """))
```

Comment:
If you want to have only 1 help text (instead of help for every field) then you should use fieldsets and place the help text at the top of the form. Then you will not need to redefine the form, see BugSystemAdmin.

Reply:
I think having only one text is better because even if we have multiple helpers, they will just lead to the documentation where everything is described in detail.

So, unless you prefer to have multiple helper text I would make it with only one as another fieldset